packages/web-components/src/components/va-icon/va-icon.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  Component,
+  Host,
+  Prop,
+  h,
+  getAssetPath
+} from '@stencil/core';
+import classnames from 'classnames';
+import { consoleDevError } from '../../utils/utils';
+
+/**
+ * @componentName Icon
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+
+@Component({
+  tag: 'va-icon',
+  styleUrl: 'va-icon.scss',
+  shadow: true,
+  assetsDirs: ['../../assets']
+})
+export class VaIcon {
+  /**
+   * the name of the icon to use
+   */
+  @Prop() icon!: string;
+
+  /**
+   * the size variant of the icon
+   */
+  @Prop() size?: number;
```

Comment:
For the docs here, can you mention that the only accepted size values are 3 through 9?

Reply:
done!